lib/cli/ui.rb

```diff
@@ -92,6 +92,22 @@ def self.fmt(input, enable_color: true)
       CLI::UI::Formatter.new(input).format(enable_color: enable_color)
     end
 
+    # Works like +CLI::UI.fmt+, but right pads with enough spaces to fill the entire terminal line.
+    # Useful for overwriting.
+    #
+    # ==== Attributes
+    #
+    # * +input+ - input to format
+    #
+    # ==== Options
+    #
+    # * +enable_color+ - should color be used? default to true
+    #
+    def self.fmt_full_line(input, enable_color: true)
```

Comment:
We should be able to accomplish the same thing by writing something like:

```
\e[nA or \e[nB # to move to the right row
\r # to reset to col 0
<text>
\e[K # to clear to the end of the line
```

This would be flicker-free too.